cime_config/testdefs/testlist_clm.xml

```diff
@@ -538,6 +538,24 @@
       <option name="wallclock">00:20:00</option>
     </options>
   </test>
+  <test name="ERP_D_P36x2_Ld3" grid="f10_f10_mg37" compset="I1850Clm51BgcCrop" testmods="clm/mimics">
```

Comment:
One thing to think about here is the length of the simulation. Since, MIMICS is going to take a long time to spinup we should likely do at least one test that runs for a few years. Or we should startup from a state where MIMICS was spunup.

Reply:
Agreed. For now the /mimics mods directory uses a cold start, but let's plan on switching to spun-up initial conditions when we have an acceptable finidat file.

Reply:
See another conversation in this PR discussing [spun up initial conditions](https://github.com/ESCOMP/CTSM/pull/1318/files/4eb33728bda7eb9bfe142d6fae20601370fba0fa#r747797589)